.github/workflows/Packages - EdFi.Db.Deploy.yml

```diff
@@ -46,31 +49,19 @@ jobs:
           path: Ed-Fi-ODS-Implementation/
     - name: Is pull request branch exists in Ed-Fi-ODS-Implementation
       working-directory: ./Ed-Fi-ODS-Implementation/
-      shell: pwsh
+      shell: powershell
```

Comment:
Why did we switch from pwsh to powershell here and in the step below?

Reply:
Fixed it with pwsh